src/components/IntegrationPrompt.tsx

```diff
@@ -27,17 +25,28 @@ npx --yes @posthog/wizard@latest --default --region <ph_region>`
             <div className="rounded border border-border dark:border-border-dark px-5 py-4 bg-white dark:bg-accent-dark">
                 <div className=" items-center gap-2 mb-1">
                     <h4 className="text-lg font-bold m-0">
-                        <span className=" mr-1">Integrate PostHog in 90 seconds with this prompt</span>
+                        <span className=" mr-1">Install in 90 seconds with this prompt</span>
                         <span className="bg-orange/10 !text-orange px-1 rounded-sm text-[13px] font-semibold flex-shrink-0 border border-orange">
-                        BETA
+                            BETA
                         </span>
                     </h4>
-                    
                 </div>
                 <p className="text-[15px] text-primary dark:text-primary-dark mb-4">
-                    Works with agent-based coding tools like Cursor and Bolt.
+                    Works with tools like{' '}
+                    <Link to="https://cursor.com/api/auth/login" external={true}>
+                        Cursor
+                    </Link>
+                    ,{' '}
+                    <Link to="https://replit.com/" external={true}>
+                        Replit
+                    </Link>{' '}
+                    and{' '}
+                    <Link to="https://bolt.new" external={true}>
+                        Bolt
+                    </Link>
```

Comment:
Also, just having the links here doesn't really solve the "Copy this prompt and go to a separate program" problem, it still feels like that. 

I was sort of expecting it to magically run in these platforms.